File: src/test/java/application/history/endpoint/history/api/GetHistoryTest.java
package application.history.endpoint.history.api;

import application.history.IntegrationTest;
import application.history.WithFaker;
import application.history.history.application.services.GetHistoryByDateService;
import lombok.SneakyThrows;
import org.junit.jupiter.api.Test;
import org.springframework.beans.factory.annotation.Autowired;


public class GetHistoryTest extends IntegrationTest implements WithFaker {

    @Autowired
    private GetHistoryByDateService service;

    @SneakyThrows
    @Test
    public void getAllHistories() {
    }
}

File: src/test/kotlin/com/example/history/application/history/endpoint/history/api/GetHistoryTest.kt
package com.example.history.application.history.endpoint.history.api

import application.history.IntegrationTest
import org.junit.jupiter.api.Test

class GetHistoryTest : IntegrationTest(), WithFaker {
    @Autowired
    private val service: GetHistoryByDateService? = null

    @get:Test
    @get:SneakyThrows
    val allHistories: Unit
        get() {}
}
